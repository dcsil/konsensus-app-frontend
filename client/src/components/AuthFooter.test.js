import React from 'react';
import { shallow } from 'enzyme';
import AuthFooter from './Footers/AuthFooter';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test AuthFooter component', () => {
  const wrapper = shallow(<AuthFooter />);
  expect(wrapper).toMatchSnapshot();
});
