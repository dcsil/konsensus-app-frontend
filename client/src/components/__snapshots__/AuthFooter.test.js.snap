// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test AuthFooter component 1`] = `ShallowWrapper {}`;
